webapp/src/components/backstage/runs_list/filters.tsx

```diff
@@ -120,6 +139,11 @@ const Filters = ({fetchParams, setFetchParams, fixedTeam}: Props) => {
         });
     }
 
+    async function fetchPlaybooks() {
+        const playbooks = await clientFetchPlaybooks(fetchParams.team_id || '', {team_id: fetchParams.team_id || '', sort: 'title'});
+        return playbooks ? playbooks.items : {} as Playbook[];
```

Comment:
Nit: can we do this instead? It feels weird to convert an empty object to a `Playbook[]`
```suggestion
        return playbooks ? playbooks.items : [] as Playbook[];
```